packages/playwright/src/matchers/expect.ts

```diff
@@ -104,11 +105,13 @@ export const printReceivedStringContainExpectedResult = (
 
 type ExpectMessage = string | { message?: string };
 
-function createMatchers(actual: unknown, info: ExpectMetaInfo): any {
-  return new Proxy(expectLibrary(actual), new ExpectMetaInfoProxyHandler(info));
+function createMatchers(actual: unknown, info: ExpectMetaInfo, prefix: string[], parentPrefixes: string[][]): any {
+  return new Proxy(expectLibrary(actual), new ExpectMetaInfoProxyHandler(info, [prefix, ...parentPrefixes]));
 }
 
-function createExpect(info: ExpectMetaInfo) {
+const getPrefixSymbol = Symbol('get prefix');
+
+function createExpect(info: ExpectMetaInfo, prefix: string[] = [], parentPrefixes: string[][] = []) {
```

Comment:
In situations like this keep all the new params required, that would reduce the subsequent regressions where the param is omitted by accident.